UltiSnips/c.snippets

```diff
@@ -96,4 +96,54 @@ snippet fund "function declaration" b
 ${1:void} ${2:function_name}($3);
 endsnippet
 
+global !p
+def split_line(text):
+	import textwrap
+	lines = textwrap.wrap(text, 78 - 19)
+	output = list()
+	for line in lines:
+		output.append('*' + ' '*19 + line)
+	snip_line = snip.tabstops[4].end[0]
+	snip.buffer.append(output, snip_line + 1)
+	del snip.buffer[snip_line]
+
+def get_args(arglist):
+	args = [arg.strip() for arg in arglist.split(',') if arg]
+	return args
+endglobal
+
+post_jump "if snip.tabstop == 0 : split_line(snip.tabstops[4].current_text)"
+snippet head "File Header" b
+/******************************************************************************
+* File:             `!p snip.rv = fn`
+*
+* Author:           ${2:`git config --get user.name`}  
+* Created:          `date +%m/%d/%y` 
+* Description:      ${4:I need a really long default here so I don't have to type so much !}
```

Comment:
Could you remove the placeholder text here please? https://github.com/honza/vim-snippets#things-to-consider-when-contributing

Thank you

Reply:
Which placeholder? the name or the Description, or both?